app/presenters/organisations/featured_news_presenter.rb

```diff
@@ -0,0 +1,48 @@
+module Organisations
+  class FeaturedNewsPresenter
+    attr_reader :org
+
+    def initialize(organisation)
+      @org = organisation
+    end
+
+    def first_featured_news
+      main_story = featured_news([org.ordered_featured_documents.first])
+      main_story = main_story[0]
+      main_story[:large] = true
+      main_story
+    end
+
+    def remaining_featured_news
+      return featured_news(org.ordered_featured_documents[1, 100]) if org.is_news_organisation?
+      all_featured_news
+    end
+
+    def all_featured_news
```

Comment:
If this is unlikely to be used anywhere else, I'd be tempted not to make this it's own separate method and simply move this line into `remaining_featured_news`.